core/modules/compare/store/mutation-types.ts

```diff
@@ -2,4 +2,4 @@ export const SN_COMPARE = 'compare'
 export const COMPARE_ADD_ITEM = SN_COMPARE + '/ADD'
 export const COMPARE_DEL_ITEM = SN_COMPARE + '/DEL'
 export const COMPARE_LOAD_COMPARE = SN_COMPARE + '/LOAD'
-export const SET_COMPARE_LOADED = `${SN_COMPARE}/SET_COMPARE_LOADED`
+export const SET_COMPARE_LOADED = SN_COMPARE + '/SET_COMPARE_LOADED'
```

Comment:
```suggestion
export const SET_COMPARE_LOADED = `${SN_COMPARE}/SET_COMPARE_LOADED`
```
why have you change to concatenation? Template literals are great ES6 feature and increase readability.

Reply:
because all of the rest types were implemented in this way so I wanted to keep consistency